addons/im_livechat/static/src/core/web/thread_model_patch.js

```diff
@@ -0,0 +1,20 @@
+import { Thread } from "@mail/core/common/thread_model";
+import { fields } from "@mail/model/misc";
+import { convertBrToLineBreak } from "@mail/utils/common/format";
+
+import { patch } from "@web/core/utils/patch";
+
+patch(Thread.prototype, {
+    setup() {
+        super.setup();
+        this.livechat_note = fields.Html();
+        this.livechatNoteText = fields.Attr(undefined, {
+            compute() {
+                if (this.livechat_note !== undefined) {
+                    return convertBrToLineBreak(this.livechat_note || "");
```

Comment:
Note: the name of this function is a bit misleading but basically it should be improved to be `convertHtmlToTextarea`.
br2nl is a first step, but it should also ideally add br after each "block" element, among other potential things.
It might be a non-issue if we properly use the editor here, and then the only other occurrence is in composer for which we want to use the editor too.